### PR TITLE
Removes boring theft objectives

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
tweak: Antagonists will no longer get the objective to steal slime extracts and plasma tanks.
add: The Captain has a swanky high-tech jetpack now. Antagonists will try to steal this one instead of *any* jetpack.
/:cl:

image

I also wanted to remove the RCD objective, but apparently it's been gone for like a year now. Huh.

The reasoning is pretty obvious. Stealing slime extracts pretty much *never* promotes interesting conflict and results in greentext inflation. In order to preserve the antagonist economy, it is imperative we stop printing easy greentext and make those tators put work into it.

Also removes the weird `special` and `number` objectives, which are defined, but not used by any antagonist and pretty boring anyway. Easy enough to code again if someone ever figures their antag needs the mission to steal an AEG. Completely backstage.

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Muh easy greentext.

File: code/game/gamemodes/steal_items.dm [hunk 1]
```diff
@@ -41,11 +41,6 @@
 	typepath = /obj/item/weapon/hand_tele
 	protected_jobs = list("Captain", "Research Director")
 
-/datum/theft_objective/jetpack
```
Comment (Fox-McCloud) [inline code/game/gamemodes/steal_items.dm]: Told you what to do with this one; could you please make it the Captain's jetpack, instead? Pushing an objective into viability is better than outright removal.

Comment (KasparoVy) [inline code/game/gamemodes/steal_items.dm]: Agreed

**Comment by Vivalas** [comment]
I used to think slime core was somewhat difficult until  I just hacked in once, started cutting up slimes and convinced some borg I was a research assistant

**Comment by Bram961** [comment]
aren't some of these used for vox raiders? (which is currently not in rotation yes but could be in the future)

**Comment by taukausanake** [comment]
@Bram961 Everyone intends to kill Vox Raiders dead. Maybe replace them with another similar gamemode but otherwise never expect it to ever come back in any form ever. It just will not happen

**Comment by FalseIncarnate** [comment]
Stealing plasma makes sense from a lore perspective: Nanotrasen is a company whose main profit is the extraction, refinement, and sale of plasma. The Syndicate straight up stealing this plasma is effectively stealing money from Nanotrasen's sales accounts.

That being said, the objective is rather easy which in turn makes it "boring". Not sure how to improve it off hand, but I don't think it should be removed and instead be spiced up a bit. It is so much more lore-friendly than objectives like "steal the NASA Voidsuit", which pretty much is a random space suit of little discernible importance to Nanotrasen (it's not like it is in a museum display, though based on its assumed age, it probably belongs in a museum...).

**Comment by FlattestGuitar** [comment]
Added the Captain jetpack from /tg/'s https://github.com/tgstation/tgstation/pull/12686.

File: code/game/objects/items/weapons/tanks/jetpack.dm [hunk 2]
```diff
@@ -109,6 +109,14 @@
 	..()
 	air_contents.oxygen = (6*ONE_ATMOSPHERE)*volume/(R_IDEAL_GAS_EQUATION*T20C)
 
+/obj/item/weapon/tank/jetpack/oxygen/captain
+	name = "Captain's jetpack"
+	desc = "A compact, lightweight jetpack containing a high amount of compressed oxygen."
+	icon_state = "jetpack-captain"
+	item_state = "jetpack-captain"
+	volume = 90
+	w_class = WEIGHT_CLASS_NORMAL
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/tanks/jetpack.dm]: Did you include the back sprite?

**Comment by FlattestGuitar** [comment]
They're there now!

**Comment by Fox-McCloud** [comment]
For now, let's keep the plasma tank and the slime extract one in; feel free to get rid of the rest.

And of course, the captain's jetpack needs to stay.

**Pull request merged**